cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -740,9 +740,9 @@ func init() {
 	Root.AddCommand(endDevicesCommand)
 }
 
-var errAddressMismatch = errors.DefineAborted("address_mismatch", "server address mismatch")
+var errAddressMismatchEndDevice = errors.DefineAborted("address_mismatch", "server address mismatch")
```

Comment:
Let’s use either the same error, or make sure that the error name reflects the scope